spec/system/searching_spec.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "managing users", type: :system do
+  let(:local_authority) { create(:local_authority, :default) }
+  let(:user) { create(:user, :assessor, local_authority: local_authority) }
+
+  let(:other_user) do
+    create(:user, :assessor, local_authority: local_authority)
+  end
+
+  let!(:planning_application1) do
+    create(:planning_application, user: user, local_authority: local_authority)
+  end
+
+  let!(:planning_application2) do
+    create(:planning_application, user: nil, local_authority: local_authority)
+  end
+
+  let!(:planning_application3) do
+    create(
+      :planning_application,
+      user: other_user,
+      local_authority: local_authority
+    )
+  end
+
+  before { sign_in(user) }
+
+  context "when users views her own planning applications" do
+    it "allows user to search planning applications by reference" do
+      visit(planning_applications_path(q: "exclude_others"))
+      click_link("All your applications")
+
+      selected_tab = find("div[class='govuk-tabs__panel']")
```

Comment:
`find("govuk-tabs__panel")` will return all the tabs with this class, whereas this will only return the tab that doesn't have the "govuk-tabs__panel___hidden" class, ie it gets the tab that is currently selected!